internal/components/components.go

```diff
@@ -185,6 +186,7 @@ func Components() (component.Factories, error) {
 		kafkareceiver.NewFactory(),
 		kafkametricsreceiver.NewFactory(),
 		k8sclusterreceiver.NewFactory(),
+		k8seventsreceiver.NewFactory(),
```

Comment:
Make sure to add this to the -releases repo as well, this file is not being used anymore other than when running `make run`.

https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

Reply:
Thanks for the information, I have raised the [PR](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/67) for the same.